scanpy/preprocessing/_scale.py

```diff
@@ -113,6 +113,13 @@ def scale(
         data, zero_center=zero_center, max_value=max_value, copy=copy, mask_obs=mask_obs
     )
 
+@numba.njit(parallel=True)
```

Comment:
```suggestion
@numba.njit(parallel=True, cache = True)
```

Please cache the kernel.